_publications/2020_vmcai_gbhr.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: publication
+# The quotes make the : possible, otherwise you can do it without quotes
+title: "Leveraging Compiler Intermediate Representation for Multi- and Cross-Language Verification"
+# YYYY_CONFSHORTNAME_PAPERSHORTNAME
+key: 2020_vmcai_gbhr
+
+# conference | journal | poster | workshop
+type: conference
+redirect_from:
+
+# The shortname is used for auto-generated titles
+shortname: VMCAI 2020
+# add a 2:1 aspect ratio (e.g., width: 400px, height: 200px) to the folder /assets/images/papers/
+image: 2020_vmcai_gbhr.png
+# add a 2:1 aspect ratio teaser figure (e.g., width: 1200px, height: 600px) to the folder /assets/images/papers/
+image_large: 2020_vmcai_gbhr_teaser.png
+
+# Authors in the "database" can be used with just the key (lastname). Others can be written properly.
+authors:
+- Jack J. Garzella
+- baranowski 
+- he
+- rakamaric
+
+journal-short: VMCAI
+year: 2020
+
+# bibentry: article | inproceedings | phdthesis | book
+bibentry: inproceedings
+bib:
+  booktitle: Proceedings of the 21st International Conference on Verification, Model Checking, and Abstract Interpretation (VMCAI)
+  series:
+  volume:
+  publisher:
+  editor:
+  pages:
+  doi:
+  address:
+  url:
+  number:
+  month:
+  note: to appear
+
+# Add things like "Best Paper Award at InfoVis 2099, selected out of 4000 submissions"
+award:
+
+# Use if this paper is linked to an internal project. This will link to the project site
+# project: upset
+
+# Use this if you have an external project website
+external-project:
+
+# The reference to the video entry
+video:
+# The reference to the preview viedo entry
+#preview-video:
+
+# the preprint
+pdf: 2020_vmcai_gbhr.pdf
+# A supplement PDF
+supplement: 
+
+# Extra supplements, such as talk slides, data sets, etc.
+supplements:
```

Comment:
It would be good to add a link to our benchmark suite into supplements.

Reply:
I put the github link in but I couldn't find it when tested it on my local machine.

Reply:
See this file for how to properly deal with supplements: https://github.com/visdesignlab/visdesignlab.github.io/blob/master/_publications/2019_eurovis_implicit-discretization.md